backend/src/controllers/pendingCourses/getPendingCourses.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from 'express';
+
+import { isError } from 'utils/typeGuards/isError';
+import { TMiddlewareCall } from 'interfaces/commonMiddleware';
+import { IUser } from 'interfaces/Ientities/Iusers';
+import { getPendingCoursesProvider } from 'db/providers/pendingCoursesProvider';
+
+const getPendingCourses = async (req: Request, res: Response, next: TMiddlewareCall) => {
+  const { id: userId } = res.locals;
+  try {
+    const user: IUser = await getPendingCoursesProvider(userId);
```

Comment:
You should better choose which way you will use typing: explicitly or implicitly. The thing is that you wrote the same pieces of code in different ways in different files. 
![image](https://user-images.githubusercontent.com/70802978/151831091-a89260ab-7df3-43d0-a679-78bfead8b3eb.png)
Let's just not use explicit typing where we can really avoid it.

Reply:
done